README.md

```diff
@@ -460,6 +460,37 @@ stub_get = stub_request(:get, "www.example.com")
 remove_request_stub(stub_get)
 ```
 
+### Reset global request counter
+
+```ruby
+stub_get = stub_request(:get, "www.example.com")
+Net::HTTP.get('www.example.com', '/')
+
+WebMock::RequestRegistry.instance.times_executed(stub_get.request_pattern)  # => 1
```

Comment:
`WebMock::RequestRegistry.instance.times_executed` is not part of the API.

perhaps it's better to show examples with `expect(stub_get).to have_been_requested.times(1)` 
and then `expect(stub_get).not_to have_been_requested` ?

Reply:
Sure, I will change this!